src/index.ts

```diff
@@ -48,8 +51,11 @@ const audioPlayer = createAudioPlayer();
 
 // Event triggered when the audio player goes idle (i.e. not playing anything)
 audioPlayer.on(AudioPlayerStatus.Idle, () => {
+  // Restore this logic to disconnect bot after it announces clips. Removing it causes the bot to stay in the channel
```

Comment:
This is a notable change

Reply:
With the commands now merged we can use `!gtfo` and `!cmere` to get the bot to move between voice channels if needed as well. Also this puts #3 on hold until we figure out exactly how we want to handle bot disconnects.